crates/ruff_workspace/src/options.rs

```diff
@@ -2942,28 +2942,30 @@ pub struct FormatOptions {
     )]
     pub indent_style: Option<IndentStyle>,
 
-    /// Configures the preferred quote character for strings. Valid options are:
-    ///
+    /// Configures the preferred quote character for strings. The recommended options are
     /// * `double` (default): Use double quotes `"`
     /// * `single`: Use single quotes `'`
-    /// * `preserve` (preview only): Keeps the existing quote character. We don't recommend using this option except for projects
-    ///    that already use a mixture of single and double quotes and can't migrate to using double or single quotes.
     ///
     /// In compliance with [PEP 8](https://peps.python.org/pep-0008/) and [PEP 257](https://peps.python.org/pep-0257/),
-    /// Ruff prefers double quotes for multiline strings and docstrings, regardless of the
+    /// Ruff prefers double quotes for triple quoted strings and docstrings, regardless of the
     /// configured quote style.
```

Comment:
"regardless of the configured quote style" has the potential to confuse since it's no longer true of `preserve`, maybe... "regardless of whether `single` or `double` is selected."?